website/docs/r/compute_instance_server.html.markdown

```diff
@@ -0,0 +1,134 @@
+---
+layout: "scaleway"
+page_title: "Scaleway: scaleway_compute_instance_server"
+sidebar_current: "docs-scaleway-resource-compute-instance-server"
+description: |-
+  Manages Scaleway Compute Instance servers.
+---
+
+# scaleway_compute_instance_server
+
+Creates and manages Scaleway Compute Instance servers. For more information, see [the documentation](https://developers.scaleway.com/en/products/instance/api/#servers-8bf7d7).
+
+## Examples
+
+### Minimal
+
+```hcl
+resource "scaleway_compute_instance_server" "web" {
+  type = "DEV1-S"
+  image_id = "f974feac-abae-4365-b988-8ec7d1cec10d"
+}
+```
+
+### With custom volumes and a public IP
+
+```hcl
+resource "scaleway_compute_instance_ip" "public_ip" {
+  server_id = "${scaleway_compute_instance_server.webserver.id}"
+}
+
+resource "scaleway_compute_instance_volume" "data" {
+  size_in_gb = 60
+}
+
+resource "scaleway_compute_instance_server" "web" {
+  type = "DEV1-L"
+  image_id = "f974feac-abae-4365-b988-8ec7d1cec10d"
+
+  tags = [ "hello", "public" ]
+
+  root_volume {
+    size_in_gb = 20
+  }
+
+  additional_volume_ids = [ "${scaleway_compute_instance_volume.data.id}" ]
+}
+```
+
+### With user data and could-init
+
+```hcl
+resource "scaleway_compute_instance_server" "web" {
+  type = "DEV1-L"
+  image_id = "f974feac-abae-4365-b988-8ec7d1cec10d"
+
+  tags = [ "web", "public" ]
+
+  user_data {
+    key = "plop"
+    value = "world"
+  }
+
+  user_data {
+    key = "xavier"
+    value = "niel"
+  }
+
+  cloud_init = file("${path.module}/cloud-init.yml")
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+- `type` - The server commercial type. You can use [this endpoint](https://api.scaleway.com/instance/v1/zones/fr-par-1/products/servers)
+to find all the available commercial types.
+
+- `image_id` - The base image of the server. You can use [this endpoint](https://api-marketplace.scaleway.com/images?page=1&per_page=100)
+to find the right local image `ID` for a given image `name` and a given `commercial_type`.
+
+- `name` - (Optional) The name of the server.
+
+- `tags` - (Optional) The tags associated with the server.
+
+- `security_group_id` - The [security group](https://developers.scaleway.com/en/products/instance/api/#security-groups-8d7f89) the server is attached to.
+
+- `root_volume` - (Optional) Root [volume](https://developers.scaleway.com/en/products/instance/api/#volumes-7e8a39) attached to the server on creation.
+   - `size_in_gb` - Size of the root volume in gigabytes.
+   To find the right size use [this endpoint](https://api.scaleway.com/instance/v1/zones/fr-par-1/products/servers) and
+   check the `volumes_constraint.{min|max}_size` (in bytes) for your `commercial_type`.
+   - `delete_on_termination` - (Defaults to `false`) Force deletion of the root volume on instance termination.
+
+!> **WARNING:** The `root_volume` must be local (`l_ssd`) therefore, updates to this field will trigger a stop/start of the server.
```

Comment:
> The `root_volume` must be local (`l_ssd`)

As of today this can't be other way, either rephrase "The `root_volume` is a local volume" or remove this part

Reply:
> updates to this field will trigger a stop/start of the server.

only update to `size_in_gb`, not on `delete_on_termination`

Reply:
I would simply say:
> Updating the root volume will trigger a stop/start of the server.